### PR TITLE
[Mobile Payments] CoD onboarding design changes

### PR DESCRIPTION
**Pull request by joshheald**
### Description

In discussions we have agreed some slight changes to the design of the Cash on Delivery prompt screen:

1. Use the `woo-customer-waiting` image for the error
2. Provide specific Learn More text with a link directly to the section of the docs which explains the use of Cash on Delivery.

Additionally, this PR addresses the non-blocking issues raised in during the review of #7499

### Testing instructions

Using a store with CoD disabled (check that it's disabled in `WP Admin > WooCommerce > Settings > Payments`)

1. Go to the hub menu
2. Tap Payments and wait for the spinner to complete
3. The onboarding incomplete footer message should appear: tap `Continue Setup`
4. The `Add Pay in Person to your checkout` onboarding screen should show

Changes:
1. Observe that the image has changed to a person leaning on a pile of boxes.
2. Check that the Learn More link opens the docs at the "Next, add a Cash on Delivery Payment Method" section.

### Screenshots

https://user-images.githubusercontent.com/2472348/185375378-9ba82345-2f97-4abc-bb3a-1d9ce1e2fe78.mp4

#### Before
before tweaks

---
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by pachlava — APPROVED** [review]
Since I've just started testing COD on iOS, I'm not fully sure which parts should already work, and what is still under construction. So, I only paid attention to the changes in question: the updated icon and `Learn more` pointing to the paragraph. Regarding this, it's good to go 🚀 

If I'll have any questions about current state of implementation in general, I'll ask them in separate 🙂

File: Storage/Storage/Model/Copiable/Models+Copiable.generated.swift [hunk 1]
```diff
@@ -61,20 +61,20 @@ extension GeneralStoreSettings {
         telemetryLastReportedTime: NullableCopiableProp<Date> = .copy,
         areSimplePaymentTaxesEnabled: CopiableProp<Bool> = .copy,
         preferredInPersonPaymentGateway: NullableCopiableProp<String> = .copy,
-        skippedCodOnboardingStep: CopiableProp<Bool> = .copy
+        skippedCashOnDeliveryOnboardingStep: CopiableProp<Bool> = .copy
```
Comment (toupper) [inline Storage/Storage/Model/Copiable/Models+Copiable.generated.swift:64]: Nice change! 😅

**Review by toupper — APPROVED** [review]

**Comment by toupper** [comment]
Code looks good to me 🎉

**Comment by joshheald** [comment]
> Since I've just started testing COD on iOS, I'm not fully sure which parts should already work, and what is still under construction. So, I only paid attention to the changes in question: the updated icon and `Learn more` pointing to the paragraph. Regarding this, it's good to go 🚀
> 
> If I'll have any questions about current state of implementation in general, I'll ask them in separate 🙂

Thanks for checking @pachlava 

To catch you up with the current state: 
- The screen should show up in onboarding if you have the feature flag on (localDeveloper and alpha builds only.)
- The screen should be skippable. If you skip the screen, it should not show again unless the app is re-installed.
- The enable button does not do anything yet. #7519 is the start of the work on the button, but it still won't do anything with that PR.

**Comment by pachlava** [comment]
> To catch you up with the current state...

Thank you, @joshheald!

**Pull request merged**